src/modals/vm-manage-gpu/GpuDataProvider.js

```diff
@@ -93,12 +77,42 @@ const GpuDataProvider = ({ children, vmId }) => {
     return selectedMdevs
   }
 
-  const isNoDisplay = (customProperties) => {
-    const parsedMdevProperties = parseMdevCustomProperty(customProperties)
-    if (parsedMdevProperties.length > 0 && parsedMdevProperties[0] === 'nodisplay') {
+  const isNoDisplay = (mdevDevices) => {
+    return isNoDisplaySpecParam(mdevDevices?.vm_mediated_device?.[0])
```

Comment:
To be consistent with backend processing, it would be better to check if any of the devices has nodisplay, rather than the first one.